index.js

```diff
@@ -130,6 +132,11 @@ function parseRubyEngineAndVersion(rubyVersion) {
     const rubyLine = toolVersions.split(/\r?\n/).filter(e => /^ruby\s/.test(e))[0]
     rubyVersion = rubyLine.match(/^ruby\s+(.+)$/)[1]
     console.log(`Using ${rubyVersion} as input from file .tool-versions`)
+  } else if (rubyVersion === 'mise.toml') { // Read from mise.toml
```

Comment:
https://mise.jdx.dev/configuration.html has 6 possible locations :sweat_smile: 
Hopefully `mise.toml` is the most common one, I don't really want to logic to search all those.
So yeah, let's look for just `mise.toml` for now.

Reply:
i see, you are right :sweat_smile: especially the conf.d would be crap :grin: